python/pyspark/ml/util.py

```diff
@@ -437,7 +437,7 @@ def extractJsonParams(instance: "Params", skipParams: Sequence[str]) -> Dict[str
     def saveMetadata(
         instance: "Params",
         path: str,
-        sc: SparkContext,
+        sparkSession: SparkSession,
```

Comment:
Same comment here about retaining compatibility

Reply:
What if I keep the same method signature, with the `sc` and inside it I use the builder to obtain the spark session object ?
```
    val sparkSession = SparkSession.builder.getOrCreate()
```